lib/cli/scripts/changelog.ts

```diff
@@ -109,7 +109,7 @@ function getCommits(options: DiffOptions): Array<Commit> {
     let log = shell.exec(command, { cwd: options.dir, silent: true }).toString();
 
     // https://stackoverflow.com/a/13928240/14644447
-    log = log.trim().replace(/"/gm, '\\"').replace(/\^@\^/gm, '"');
+    log = JSON.stringify(log.trim()).slice(1, -1).replace(/\^@\^/gm, '"');
```

Comment:
i'm not sure it is going to work as log is not an object so you can't stringify it

Reply:
`log` is a string, we use `JSON.stringity` to escape the quotes

Reply:
did you test the changelog command? does it produce the expected output?

Reply:
Added a fix, now works as expected